clang/lib/Headers/float.h

```diff
@@ -47,6 +47,8 @@
     (defined(__cplusplus) && __cplusplus >= 201103L) ||                        \
     (__STDC_HOSTED__ && defined(_AIX) && defined(_ALL_SOURCE))
 #    undef DECIMAL_DIG
+#    undef INFINITY
```

Comment:
The define guarding the undefs needs to be the same as the one for the defines, I think?

Reply:
Good catch, I've corrected this.